index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
- Great job adding a .html file :muscle:

Reply:
Thank You